app/src/main/res/values/strings.xml

```diff
@@ -14,8 +13,18 @@
     <string name="genres_rock">Rock</string>
     <string name="genres_edm">EDM</string>
     <string name="genres_electronic">Electronic</string>
-
     <string name="image_track">image_track</string>
     <string name="image_more_info">moreinfo</string>
+    <string name="add_album">Add Album</string>
+    <string name="name_album">Name Album</string>
+    <string name="err_name_exist">Album name  is exist !!</string>
+    <string name="err_name_null">Name album is empty</string>
+    <string name="song">Songs</string>
+    <string name="delete">Delete</string>
+    <string name="rename">Rename</string>
+    <string name="rename_album">Rename Album</string>
+    <string name="ok">Ok</string>
```

Comment:
WARNING: [ButtonCase] The standard Android way to capitalize Ok is "OK" (tip: use `@android:string/ok` instead)
 The standard capitalization for OK/Cancel dialogs is "OK" and "Cancel". To ensure that your dialogs use the standard strings, you can use the resource strings @android:string/ok and @android:string/cancel.